### PR TITLE
fix: heap out of memory error

### PR DESCRIPTION
**Pull request by craicoverflow**
See #1415 

This PR was to fix an out of memory error in CI. The initial fix in https://github.com/aerogear/graphback/commit/4a16de7ea20accb8daf0b4922d95e862ab999770 increased the memory available to the CI but we wanted to see if we could fix this locally too without having to increase memory.

I am now seeing errors in the Mongo tests - it is related to thew tests timing out waiting for MongoMemoryServer to download.

I have always seen this error locally, but it was never an issue on CI until now. @wtrocki @ssd71 did not experience this as far as I am aware.

I tried resolving it by ensuring the MongoMemoryServer only gets downloaded once per test (see https://github.com/aerogear/graphback/pull/1420/commits/6f8556cc211fc103b1c6db109d91c4b4bcac798e), because it was being re-downloaded in every test.

Help/ideas appreciated @ssd71 @machi1990 @wtrocki

### CODE REVIEW

**Comment by machi1990** [comment]
@craicoverflow amazing job.I'lll have to try this in my local machine. 

Look like there are some conflicts, can you do a rebase?

**Comment by craicoverflow** [comment]
> @craicoverflow amazing job.I'lll have to try this in my local machine.
> 
> Look like there are some conflicts, can you do a rebase?

Done.

File: packages/graphback-cli/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "lint-staged": "10.2.9",
     "ncp": "2.0.0",
     "rimraf": "3.0.2",
-    "ts-jest": "25.5.1",
+    "ts-jest": "26.0.1",
```
Comment (machi1990) [inline packages/graphback-cli/package.json:38]: Looks like `26.0.1` is not there, should be `26.1.0`. 

I had to manually pick the version locally after yarn proposed it to me. 

Also CI is failing for similar reason 
```
Error: Couldn't find any versions for "ts-jest" that matches "26.0.1"
 
    at MessageError.ExtendableBuiltin (/opt/yarn-v1.22.4/lib/cli.js:721:66)
 
    at new MessageError (/opt/yarn-v1.22.4/lib/cli.js:750:123)
 
    at Function.<anonymous> (/opt/yarn-v1.22.4/lib/cli.js:50545:13)
 
    at Generator.next (<anonymous>)
 
    at step (/opt/yarn-v1.22.4/lib/cli.js:310:30)
 
    at /opt/yarn-v1.22.4/lib/cli.js:321:13
 
    at runMicrotasks (<anonymous>)
 
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
 
``

**Comment by machi1990** [comment]
Verified this locally a couple of times and did not get the error.

**Comment by ssd71** [comment]
> I am now seeing errors in the Mongo tests - it is related to thew tests timing out waiting for MongoMemoryServer to download.
> 
> I have always seen this error locally, but it was never an issue on CI until now. @wtrocki @ssd71 did not experience this as far as I am aware.
> 
> I tried resolving it by ensuring the MongoMemoryServer only gets downloaded once per test (see [6f8556c](https://github.com/aerogear/graphback/commit/6f8556cc211fc103b1c6db109d91c4b4bcac798e)), because it was being re-downloaded in every test.

I actually did run into this in the past week, but wasn't sure if it was repoducible or not, just assumed it was bandwidth problem. Since then I have been using the jest extension for vscode so haven't run into this after that.

**Pull request merged**